Hestia/app/src/main/java/hestia/backend/DeviceListRetrieverTask.java

```diff
@@ -40,7 +40,7 @@ public DeviceListRetrieverTask() {
      */
     @Override
     protected ArrayList<Device> doInBackground(Void... voids) {
-        String devicesPath = this.cic.getPath() + "devices/";
+        String devicesPath = this.backendInteractor.getPath() + "devices/";
```

Comment:
especially endpoints are important to not have hard-coded, as they are likely to change